test/README.md

```diff
@@ -21,7 +21,7 @@ And depending on which PATTERN is chosen, a series of workload containers
 - Install make and jq
   - `apt update && apt install -y make jq`
 - Install golang version 1.10.* ...
```

Comment:
Need to change the version in the text above

Reply:
done